doc/manual/cookbooks/benchmarks/burstedde/burstedde.prm

```diff
@@ -1,4 +1,3 @@
-set Linear solver tolerance                = 1e-12
```

Comment:
This is not the default value, so you can not just remove it. Please add it below to the Stokes solver parameters.